util/src/main/java/tc/oc/pgm/util/compatability/SportPaperListener.java

```diff
@@ -0,0 +1,127 @@
+package tc.oc.pgm.util.compatability;
+
+import java.util.EnumSet;
+import java.util.Set;
+import org.bukkit.Bukkit;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import tc.oc.pgm.util.event.sport.block.BlockDispenseEntityEvent;
+import tc.oc.pgm.util.event.sport.block.BlockFallEvent;
+import tc.oc.pgm.util.event.sport.entity.EntityDespawnInVoidEvent;
+import tc.oc.pgm.util.event.sport.entity.EntityExtinguishEvent;
+import tc.oc.pgm.util.event.sport.entity.ExplosionPrimeByEntityEvent;
+import tc.oc.pgm.util.event.sport.entity.PotionEffectRemoveEvent;
+import tc.oc.pgm.util.event.sport.player.PlayerAttackEntityEvent;
+import tc.oc.pgm.util.event.sport.player.PlayerLocaleChangeEvent;
+import tc.oc.pgm.util.event.sport.player.PlayerOnGroundEvent;
+import tc.oc.pgm.util.event.sport.player.PlayerSkinPartsChangeEvent;
+import tc.oc.pgm.util.event.sport.player.PlayerSpawnEntityEvent;
+import tc.oc.pgm.util.skin.Skin;
+
+public class SportPaperListener implements Listener {
+
+  private static void handleCall(Event pgmEvent, Event sportEvent) {
+    if (sportEvent instanceof Cancellable) {
+      ((Cancellable) pgmEvent).setCancelled(((Cancellable) sportEvent).isCancelled());
+      Bukkit.getServer().getPluginManager().callEvent(pgmEvent);
+      ((Cancellable) sportEvent).setCancelled(((Cancellable) pgmEvent).isCancelled());
+    } else {
+      Bukkit.getServer().getPluginManager().callEvent(pgmEvent);
+    }
+  }
+
+  @EventHandler(ignoreCancelled = true)
```

Comment:
Does the event priority matter here?

Reply:
Not unless another plugin is handling SportPaper specific events